ansible/roles/docker/defaults/main.yml

```diff
@@ -2,3 +2,4 @@
 docker_debian_version: "{{ docker.debian.version if docker_debian_version_defined else '5:18.09.1~3-0~ubuntu-bionic' }}"
 docker_redhat_version: "{{ docker.redhat.version if docker_redhat_version_defined else '17.03.2.ce' }}"
 docker_logging_max_size: 100m
+docker_xenial_version: "17.03.2~ce-0~ubuntu-xenial"
```

Comment:
Should this version be updated? I believe this version is vulnerable to CVE-2019-5736: https://blog.docker.com/2019/02/docker-security-update-cve-2018-5736-and-container-security-best-practices/

Reply:
Yes, you're right, I should go update it. Thanks for catching that. I'll change it to 18.06.2 (assuming that will install on 16.04) and push up those changes.

Reply:
Fixed via 37329d6.